docs/marketplace/jenkins/enterprise-store/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Setting Up Appcircle Enterprise Store Plugin
+sidebar_label: Enterprise App Store
+description: Enhance powerful plugin to publish your builds to appcircle app store
+tags:
+  [
+    overview,
+    concepts,
+    app store,
+    internal-testing,
+    beta testing,
+    binary distribution,
+    ipa distribution,
+    apk distribution,
+  ]
+sidebar_position: 2
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+The Appcircle Enterprise Store plugin allows users to publish their apps to appcircle app store
+
+### Install Appcircle Enterprise Store Plugin
+
+Go to your Jenkins dashboard and follow, Manage Jenkins > Plugins and search for "Appcircle Enterprise Store"
```

Comment:
Go to your Jenkins dashboard and navigate to Manage Jenkins > Manage Plugins. Then, search for "Appcircle Enterprise Store" in the available plugins section.